MAUI/Cards/getting-started.md

```diff
@@ -7,19 +7,73 @@ control: SfCard
 documentation: ug
 ---
 
-# Getting started with .NET MAUI Card (SfCard)
-This section details the process of integrating the Card control and focuses solely on the fundamental features required for initiating your exploration of Syncfusion Card.
+# Getting started with .NET MAUI Card
+This section details the process of integrating the [.NET MAUI Cards](https://www.syncfusion.com/maui-controls/maui-cards) control and focuses solely on the fundamental features required for initiating your exploration of Syncfusion Card. Follow the steps below to add a basic cards view to your project.
 
+## Prerequisites
 
-## Creating an application using the .NET MAUI Card
+Before proceeding, ensure the following are set up:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
 
-1. Create a new .NET MAUI application in Visual Studio.
+## Step 1: Create a New MAUI Project
 
-2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfCards` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Cards](), and then install.
+### Visual Studio
 
-3. To initialize the control, import the control namespace `Syncfusion.Maui.Cards` in XAML or C# code.
+1. Go to **File > New > Project** and choose the **.NET MAUI App** template.
+2. Name the project and choose a location, then click **Next**.
+3. Select the .NET framework version and click **Create**.
 
-4. Initialize `SfCards.`
+### Visual Studio Code
+
+1. Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+4. Then choose **Create project.**
+
+## Step 2: Install the Syncfusion MAUI ProgressBar NuGet Package
```

Comment:
mentioned it as progress bar- change it to cards

Reply:
Changed control name